.github/workflows/integration_test.yml

```diff
@@ -0,0 +1,39 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run yarn install
+        run: yarn install
+
+      - name: Download encointer-node
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: build-and-test.yml
+          name: executables
+          # in fact this action should download the latest artifact, but sometimes fails. Then we need to
+          # set the `run_id` to force a download of an updated binary.
+          run_id: 1282695776
```

Comment:
artifacts arw not persisted for very long. we can't rely on runs. can we use the latest release instead?

Reply:
We can leave it until we run into issues ;-) It's done in other repos too anyway